### PR TITLE
ci: add more auto-labels

### PR DESCRIPTION
**Pull request by oklopfer**

### CODE REVIEW

File: .github/labeler.yml [hunk 1]
```diff
@@ -1,8 +1,35 @@
...
+- label: "chore"
+  title: "^chore:.*"
+- label: "CI"
+  title: "^ci:.*"
+- label: "help wanted"
+  title: "^help:.*"
+- label: "meta"
+  title: "^meta:.*"
+- label: "package request"
+  title: "^req:.*"
```
Comment (Henryws) [inline .github/labeler.yml:31]:
```suggestion
```
This is for issues, not PRs.

File: .github/labeler.yml [hunk 2]
```diff
@@ -1,8 +1,35 @@
...
+- label: "revert"
+  title: '^rev\(.*'
+
+# type 2: repo-action labels
 - label: "Package Add"
   title: "^add:.*"
 - label: "remove package"
   title: "^rm:.*"
+- label: "dependencies"
+  title: "^dep:.*"
```
Comment (Henryws) [inline .github/labeler.yml:21]:
```suggestion
```
We use dependabot to update pre-commit, and it handles tags for us.

File: .github/labeler.yml [hunk 3]
```diff
@@ -1,8 +1,35 @@
...
+- label: "chore"
+  title: "^chore:.*"
+- label: "CI"
+  title: "^ci:.*"
+- label: "help wanted"
+  title: "^help:.*"
```
Comment (Henryws) [inline .github/labeler.yml:27]:
```suggestion
```
PRs aren't where help wanted things go.

File: .github/labeler.yml [hunk 4]
```diff
@@ -1,8 +1,35 @@
...
+- label: "meta"
+  title: "^meta:.*"
+- label: "package request"
+  title: "^req:.*"
+- label: "security"
+  title: "^sec:.*"
+- label: "question"
+  title: "^qa:.*"
```
Comment (Henryws) [inline .github/labeler.yml:35]:
```suggestion
```

**Review by Henryws — CHANGES_REQUESTED** [review]

File: .github/labeler.yml [hunk 5]
```diff
@@ -1,8 +1,27 @@
...
 - label: "Package Add"
   title: "^add:.*"
 - label: "remove package"
   title: "^rm:.*"
+- label: "chore"
+  title: "^chore:.*"
+- label: "CI"
+  title: "^ci:.*"
+- label: "meta"
+  title: "^meta:.*"
```
Comment (Henryws) [inline .github/labeler.yml:25]:
```suggestion
```
What would a meta PR be for?

File: .github/labeler.yml [hunk 6]
```diff
@@ -1,8 +1,27 @@
...
+- label: "security"
+  title: "^sec:.*"
```
Comment (Henryws) [inline .github/labeler.yml:27]:
```suggestion
```
If the security fix is in the pacscript, maintainers would use fix, and if it's an upstream fix, it'd be an upd.

**Review by Henryws — CHANGES_REQUESTED** [review]

File: .github/labeler.yml (hunk 5 above)
Comment (oklopfer) [reply .github/labeler.yml:25]: an overall change to the pacstall-programs repo or to how pacstall functions

Comment (Henryws) [reply .github/labeler.yml:25]: Then wouldn't that be a feat?

Comment (Henryws) [reply .github/labeler.yml:25]: Then I'll make one.

Comment (oklopfer) [reply .github/labeler.yml:25]: I can swap this to `feat` when you do

File: .github/labeler.yml [hunk 7]
```diff
@@ -1,8 +1,25 @@
...
 labels:
+# type 1: package-action labels
 - label: "Update Package"
   title: '^upd\(.*'
+- label: "Update Pacscript"
+  title: '^fix\(.*'
+- label: "bug"
+  title: '^bug\(.*'
+- label: "enhancement"
+  title: '^enh\(.*'
```
Comment (Henryws) [inline .github/labeler.yml:11]:
```suggestion
```
Enhancements are features for pacscripts.

**Review by Henryws — CHANGES_REQUESTED** [review]

File: .github/labeler.yml [hunk 8]
```diff
@@ -1,8 +1,23 @@
...
 labels:
+# type 1: package-action labels
 - label: "Update Package"
   title: '^upd\(.*'
+- label: "Update Pacscript"
+  title: '^fix\(.*'
+- label: "bug"
+  title: '^bug\(.*'
+- label: "revert"
+  title: '^rev\(.*'
```
Comment (Henryws) [inline .github/labeler.yml:11]:
```suggestion
```

**Review by Henryws — CHANGES_REQUESTED** [review]

**Review by Henryws — APPROVED** [review]

**Pull request merged**